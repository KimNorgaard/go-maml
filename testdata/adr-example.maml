{
  project: "MAML"
  tags: [
    "minimal"
    "readable"
  ]

  # A simple nested object
  spec: {
    version: 1
    author: "Anton Medvedev"
  }
}
