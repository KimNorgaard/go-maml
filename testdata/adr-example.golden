{
  project: "MAML",
  tags: [
    "minimal",
    "readable"
  ],
  spec: {
    version: 1,
    author: "Anton Medvedev"
  }
}